tests/test_mwp.py

```diff
@@ -0,0 +1,30 @@
+from happytransformer import HappyWordPrediction
+
+
+def test_mwp_basic():
+    happy_mwp = HappyWordPrediction()
+    result = happy_mwp.predict_mask(
+        "Please pass the salt and [MASK]",
+    )
+    answer = {'score': 0.2664579749107361, 'token_str': 'pepper'}
```

Comment:
if we're rewriting for v2 can we use named tuples?

Reply:
yes good call